doc/api/examples/map.json

```diff
@@ -5,6 +5,7 @@
         "desc": "Example map for the API",
         "permission": "commons",
         "screenshot": "https://s3.amazonaws.com/metamaps-assets/site/missing-map.png",
+        "starred": false,
```

Comment:
yeye. Ok I am happy to go that direction, with the background you've provided :) leave as is